avm/src/vm/opcode.rs

```diff
@@ -3838,6 +3838,323 @@ pub static OPCODES: Lazy<HashMap<i64, ByteOpcode>> = Lazy::new(|| {
       Ok(hand_mem)
     })
   });
+  unpred_cpu!(dsrrun => fn(args, mut hand_mem) {
```

Comment:
these opcodes can just call the functions in `daemon::ctrl` right?

Reply:
No, because the control port itself doesn't exist when you `alan run foo.agz` instead of `alan deploy new foo.agz`.